### DISCUSSION TITLE
Adding a program to "winget" or "choko"

### DISCUSSION THREAD

**Discussion by Sarvensky** [ErrorFlynn/ytdlp-interface#138] [General]
Dear developer, thank you for your hard work and useful program. But is it possible to make it possible to install from "winget" or "choko" directory for convenience?

**Comment by ErrorFlynn** [comment]
I took a brief look at what it would take to make the program available through those package managers, and it seems like way too much trouble right now. Maybe I'll do it at some point, but I just don't have the time and energy right now.
